packages/web/src/components/collection/desktop/Artwork.tsx

```diff
@@ -62,7 +62,7 @@ export const Artwork = (props: ArtworkProps) => {
 
   useEffect(() => {
     // If there's a gradient, this is a smart collection. Just immediately call back
-    if (image || gradient || imageOverride || image === '') callback()
```

Comment:
maybe this needed? since this is separate from ^

Reply:
oh wow yup thanks